server/deleteMainIndex.js

```diff
@@ -0,0 +1,16 @@
+const elasticsearch = require('elasticsearch');
+
+const client = new elasticsearch.Client({
+    hosts: [ 'http://localhost:9200']
+});
+
+client.indices.delete({
+    index: 'symptoms-icare-default'
+  }, function(error, response, status) {
+    if (error) {
+        console.log(error);
+    } else {
+        console.log("deleted index", response);
+    }
+  });
+  
```

Comment:
Temporarily we would this approach to manually update the index setting in Elasticsearch. I would take a look on this issue in spring break and try to get a better approach if possible.

Reply:
Yes, this should be fine for now though.